### DISCUSSION TITLE
Refresh rate problem

### DISCUSSION THREAD

**Discussion by joaovs967** [ppy/osu#16333] [Q&A] [answered]
i have a 60hz screen running at 85hz (custom resolution at nvidia control panel) so 1080p@85hz is the default desktop resolution and all my apps runs at this refresh rate, but when i open osu!lazer it makes the refresh rate stuck at 1080p@60hz, ignoring the higher refresh rate available, and either "default" and "1920x1080" has no effect,  the screen stays at 1080p@60hz.

is there any way to make osu!lazer stop ignoring the custom resolution and run at 85hz?

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu-framework/issues/4081
